From 9eeeefd5c9bd94082f0b0f0c41fab00ecc4a3c2e Mon Sep 17 00:00:00 2001
From: Vitaly Margolin <vmargolin@habana.ai>
Date: Tue, 17 Jun 2025 16:09:58 +0300
Subject: [PATCH] [SW-232548] lfs: add component name to LFS log init

Add componont name to LFS logger initialization
This is done to simplify robot testing of LFS logger
feature

Change-Id: I514417e43a9ec65ecb013d4bfaf2d7d2bc8cf2b8
Signed-off-by: Vitaly Margolin <vmargolin@habana.ai>
---

diff --git a/fw-loader/src/main.c b/fw-loader/src/main.c
index ed1a31c..04b0694 100644
--- a/fw-loader/src/main.c
+++ b/fw-loader/src/main.c
@@ -206,7 +206,7 @@
 #ifdef CONFIG_LFS_ENABLED
 static void start_lfs_logger(void)
 {
-	lfs_console_init_time_critical();
+	lfs_console_init_time_critical_by_component("FW Loader Agent");
 }
 #else
 static void start_lfs_logger(void) {}
